omicron/cli/process.py

```diff
@@ -114,6 +114,49 @@ def gps2str(gps):
     return ret
 
 
+def clean_dirs(dir_list):
+    """Remove any empty directories we created"""
+    for adir in dir_list:
+        pdir = Path(adir)
+        flist = list(pdir.glob('*'))
+        if len(flist) == 0:
+            pdir.rmdir()
+        else:
+            can_delete = True
+            for file in flist:
+                if file.is_dir():
+                    can_delete = remove_empty_dir(file)
+                else:
+                    can_delete = False
```

Comment:
@areeda Do you also want to a `break` here so that once `can_delete = False`, we know that we don't need to keep trying

Reply:
This was designed with the idea it that it would be run if there are no segments to process and after DAG completes. Only the first one is implemented in this PR to get us to the SCCB. For the second, inside the trigger (omicron output)directory and merge (collapsed contiguous trigger files) have a directory for each channel in the group. Some may be empty while the problem channels will have intermediate files.

Reply:
I don't quite follow (perhaps my coffee has not kicked in yet). What is the "first one" and "second"? My comment (I think) is just to add a `break` after the `can_delete = False` line. It just saves a bit of looping in case a file is found

Reply:
Ok, I understand now that there is some recursion happening within `remove_empty_dir()` so we cannot add the `break` statement.